version.js

```diff
@@ -208,9 +223,9 @@ function checkAll(versions, source, dep) {
 	const devDeps = source.devDependencies;
 	const deps = source.dependencies;
 	const peer = source.peerDependencies;
-	const isModDevDeps = check(devDeps, dep, version);
+	const isModDevDeps = check(devDeps, dep, version, 'dev');
```

Comment:
i don't get the name isModDevDeps is not a function shouldn't it be updatedDependencies or something like that

also check seem to only return a boolean. it is confusing